llvm/test/CodeGen/SPIRV/CombinedSamplerImageDynIdx.ll

```diff
@@ -0,0 +1,40 @@
+; RUN: llc -verify-machineinstrs -O0 -mtriple=spirv1.5-vulkan-library %s -o - | FileCheck %s
```

Comment:
There is a lot of related test cases, all added to the root test folder of SPIRV, so would it be a good idea to create a separate folder for such test cases?

Reply:
Done.